CefSharp/Internals/JavascriptObjectRepository.cs

```diff
@@ -35,14 +35,15 @@ internal JavascriptObject CreateJavascriptObject()
             return result;
         }
 
-        public void Register(string name, object value)
+        public void Register(string name, object value) {  Register(name, value, true); }
```

Comment:
Remove unused method signature. As there's not way to call this publicly, the overload here is not required.
(Also the formatting is out, need to follow existing styling in general)